code/modules/power/apc.dm

```diff
@@ -120,6 +121,12 @@
 
 	var/datum/effect_system/sparks/spark_system
 
+	component_types = list(

```

Comment:
Aren't these dropped on deconstruction, but not required for construction? Won't this allow for infinite power_control boards, capacitors, and console_screens?